opencti-platform/opencti-front/src/private/components/common/form/StatusField.jsx

```diff
@@ -98,7 +98,7 @@ class StatusField extends Component {
 
   searchStatuses() {
     fetchQuery(statusFieldStatusesSearchQuery, {
-      first: 10,
+      first: 100,
```

Comment:
Reading the original issue, I'm not sure it's the source of the problem. It's not related to the status being _used_ to be queriable?

Reply:
we want to list all entity workflow statuses, whether in use or not. which is the case. in `ToolBar.jsx` we have a similar use of the `statusFieldStatusesSearchQuery`

Reply:
tested with the playbook given in the issue, that do not fix the problem.

Reply:
after discussions: yes it fixes the issue but it seems there is an other issue more UI related, cf thread of the issue.

Reply:
after discussion with @Jipegien , this fixes the bug but the UI needs to be improved by grouping status by entity type, which was done by @frapuks in this [pr](https://github.com/OpenCTI-Platform/opencti/pull/6707)